sdk/eventhub/ci.legacy.yml

```diff
@@ -0,0 +1,55 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
+  branches:
+    include:
+    - master
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/eventhub/
+    exclude:
+    - sdk/eventhub/Azure.Messaging.EventHubs
+    - sdk/eventhub/Azure.Messaging.EventHubs.Processor
+    - sdk/eventhub/Azure.Messaging.EventHubs.Shared
+    - sdk/eventhub/Azure.ResourceManager.EventHubs
+    - sdk/eventhub/Microsoft.Azure.Management.EventHub
+    - sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs
+
+pr:
+  branches:
+    include:
+    - master
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/eventhub/
+    exclude:
+    - sdk/eventhub/Azure.Messaging.EventHubs
+    - sdk/eventhub/Azure.Messaging.EventHubs.Processor
+    - sdk/eventhub/Azure.Messaging.EventHubs.Shared
+    - sdk/eventhub/Azure.ResourceManager.EventHubs
+    - sdk/eventhub/Microsoft.Azure.Management.EventHub
+    - sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs
+
+extends:
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: eventhub
+    ExcludeProjects:
+      - Azure.ResourceManager.EventHubs
+      - Azure.Messaging.EventHubs
+      - Azure.Messaging.EventHubs.Processor
+      - Azure.Messaging.EventHubs.Shared
+      - Azure.Messaging.EventHubs.*
```

Comment:
The wildcard *should* be sufficient, but there's one outstanding observation from the initial round of testing where it's not 100% confirmed for one of our script calls.   

My approach here was to explicitly define everything that exists today and leave the wild card to provide some level of filtering for anything new.  I'm still working with members of the engineering team to investigate and will open a new PR to trim to just the wild card, if possible.